pkg/cluster/config/types.go

```diff
@@ -102,4 +102,7 @@ type Networking struct {
 	// If DisableDefaultCNI is true, kind will not install the default CNI setup.
 	// Instead the user should install their own CNI after creating the cluster.
 	DisableDefaultCNI bool
+	// BridgeName is the bridge network name.
+	// Defaults to same as cluster name.
+	BridgeName string
```

Comment:
https://github.com/kubernetes-sigs/kind/issues/273#issuecomment-488874160

Reply:
(this config field name is not suitable, we need to separate docker specific functionality and make that clear)